src/app/features/StepDefinitionEditor/Models/MockDataInstanceModel.js

```diff
@@ -46,6 +46,11 @@ var createMockDataInstanceModelConstructor = function (
                 get: function () {
                     return toAST.call(this);
                 }
+            },
+            link: {
```

Comment:
pulled this one out of this pull request.Will do more analysis and submit changes in later pull request.